src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseTypeService.java

```diff
@@ -56,6 +59,7 @@ public void validateData(final Map<String, JsonNode> data,
                              final CaseType caseType) {
         final List<ValidationResult> dataValidationResults = caseDataValidator.validate(data, caseType.getCaseFields());
         if (!dataValidationResults.isEmpty()) {
+            LOG.warn("There have been validation errors={}", dataValidationResults);
```

Comment:
what kind of validation errors will be logging as WARN? things like invalid email?

Reply:
I guess yes all sorts of field validation errors.